dipy/info.py

```diff
@@ -108,3 +108,23 @@
                        "nibabel (>=%s)" % NIBABEL_MIN_VERSION,
                        "h5py (>=%s)" % H5PY_MIN_VERSION,
                        "packaging (>=%s)" % PACKAGING_MIN_VERSION]
+EXTRAS_REQUIRE = {
+    "doc": [
+        "fury>=0.6",
+        "sphinx",
```

Comment:
For the doc, there is more: you can look at: https://github.com/dipy/dipy/blob/master/doc/doc-requirements.txt